### PR TITLE
Rename the project ExdiGdbSrvSample to ExdiGdbSrv

### PR DESCRIPTION
**Pull request by alexsmendez**
This PR removed all instances of *Sample* pattern from the project.

### CODE REVIEW

File: Exdi/exdigdbsrv/yaml/Continuous-Integration.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+trigger:
+- master
+
+pool:
+  vmImage: 'windows-latest'
+
+variables:
+  buildConfiguration: 'Release'
+  major: 1
+  minor: 0
```
Comment (TimMisiak) [inline Exdi/exdigdbsrv/yaml/Continuous-Integration.yml:13]: Please update the minor version to 1, since the revision counter will get reset with this change.

**Review by TimMisiak — CHANGES_REQUESTED** [review]

File: Exdi/exdigdbsrv/ExdiGdbSrv/ReadMe.txt [hunk 2]
```diff
@@ -0,0 +1,83 @@
+========================================================================
```
Comment (TimMisiak) [inline Exdi/exdigdbsrv/ExdiGdbSrv/ReadMe.txt:1]: There are a bunch of instances here not changed from "sample". I'd say just delete the file.

File: Exdi/exdigdbsrv/ExdiGdbSrv/stdafx.cpp [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// stdafx.cpp : source file that includes just the standard includes
+// ExdiGdbSrvSample.pch will be the pre-compiled header
```
Comment (TimMisiak) [inline Exdi/exdigdbsrv/ExdiGdbSrv/stdafx.cpp:2]: Old file name.

**Review by TimMisiak — CHANGES_REQUESTED** [review]
It looks like the old files were not deleted. Can we make sure those are deleted?

File: Exdi/exdigdbsrv/yaml/Continuous-Integration.yml (hunk 1 above)
Comment (alexsmendez) [reply Exdi/exdigdbsrv/yaml/Continuous-Integration.yml]: done.

File: Exdi/exdigdbsrv/ExdiGdbSrv/ReadMe.txt (hunk 2 above)
Comment (alexsmendez) [reply Exdi/exdigdbsrv/ExdiGdbSrv/ReadMe.txt:1]: Yes, the readme.txt can be deleted

File: Exdi/exdigdbsrv/ExdiGdbSrv/stdafx.cpp (hunk 3 above)
Comment (alexsmendez) [reply Exdi/exdigdbsrv/ExdiGdbSrv/stdafx.cpp:2]: do you mean removing the stdafx.cpp file and remove it from the project as well?

Comment (alexsmendez) [reply Exdi/exdigdbsrv/ExdiGdbSrv/stdafx.cpp]: fixed

File: Exdi/exdigdbsrv/ExdiGdbSrv/LiveExdiGdbSrvServer.cpp [hunk 4]
```diff
@@ -1630,7 +1630,7 @@ HRESULT CLiveExdiGdbSrvSampleServer::SetGdbServerParameters()
         if (fileNameLength == 0)
         {
             MessageBox(0, _T("Error: the EXDI_GDBSRV_XML_CONFIG_FILE environment variable is not defined.\n")
-                          _T("The Exdi-GdbServer sample won't continue at this point.\n")
+                          _T("The Exdi-GdbServer  won't continue at this point.\n")
```
Comment (TimMisiak) [inline Exdi/exdigdbsrv/ExdiGdbSrv/LiveExdiGdbSrvServer.cpp:1633]: Nit: The message got a little messed up.

File: Exdi/exdigdbsrv/ExdiGdbSrv/LiveExdiGdbSrvServer.cpp [hunk 5]
```diff
@@ -1861,6 +1861,14 @@ ADDRESS_TYPE CLiveExdiGdbSrvSampleServer::ParseAsynchronousCommandResult(_Out_ D
                     stopReply.currentAddress = m_lastPcAddress;
                     *pProcessorNumberOfLastEvent = pController->GetLastKnownActiveCpu();
                     isWaitingOnStopReply = false;
+                } 
+                // Is it an "OK" response w/o any other field (e.g. OpenOCD can send "OK" after 's'/'g')?
+                else if (stopReply.status.isCoreRunning)
```
Comment (TimMisiak) [inline Exdi/exdigdbsrv/ExdiGdbSrv/LiveExdiGdbSrvServer.cpp:1866]: Is this change intentional? Can we take this out and apply this in its own change?

File: Exdi/exdigdbsrv/ExdiGdbSrv/sources [hunk 6]
```diff
@@ -0,0 +1,63 @@
+MSC_WARNING_LEVEL=/W4 /WX
```
Comment (TimMisiak) [inline Exdi/exdigdbsrv/ExdiGdbSrv/sources:1]: This file should be deleted, but I'm ok if we do that in a separate PR.

File: Exdi/exdigdbsrv/GdbSrvControllerLib/AsynchronousGdbSrvController.cpp [hunk 7]
```diff
@@ -702,6 +702,13 @@ void AsynchronousGdbSrvController::HandleStopReply(_In_ const std::string reply,
         *pEventNotification = true;
         *pProcessorNumber = GdbSrvController::GetLastKnownActiveCpu();
     }
+    // Is it an "OK" response w/o any other field (e.g. OpenOCD can send this after 's'/'g')?
+    else if (stopReply.status.isCoreRunning)
```
Comment (TimMisiak) [inline Exdi/exdigdbsrv/GdbSrvControllerLib/AsynchronousGdbSrvController.cpp:706]: This is another change unrelated to the rename. This should be applied separately

File: Exdi/exdigdbsrv/GdbSrvControllerLib/sources [hunk 8]
```diff
@@ -29,15 +29,15 @@ SOURCES=\
 
 INCLUDES = \
     ..\;\
-    $(OBJ_PATH)\..\exdigdbsrvsample\$(O);\
```
Comment (TimMisiak) [inline Exdi/exdigdbsrv/GdbSrvControllerLib/sources:32]: This file should be deleted, but I'm ok if we do that in a different PR.

File: Exdi/exdigdbsrv/dirs [hunk 9]
```diff
@@ -0,0 +1,2 @@
+DIRS= \
```
Comment (TimMisiak) [inline Exdi/exdigdbsrv/dirs:1]: This file should be deleted, although that can be in another PR

**Review by TimMisiak — CHANGES_REQUESTED** [review]

File: Exdi/exdigdbsrv/GdbSrvControllerLib/AsynchronousGdbSrvController.cpp (hunk 7 above)
Comment (alexsmendez) [reply Exdi/exdigdbsrv/GdbSrvControllerLib/AsynchronousGdbSrvController.cpp:706]: Ok, I'll add it separately, I have this pending in my sand box related to Ampere folks changes.
